src/desktop/components/react/stitch_components/index.tsx

```diff
@@ -1,10 +1,19 @@
 export { SearchBar } from "./SearchBar"
-export { NavBar } from './NavBar'
+export { NavBar } from "./NavBar"
+export { NavBarTest } from "./NavBarTest"
 
-export { ReactionArtworkArtistInfo as ArtworkArtistInfo } from "./ReactionArtworkArtistInfo"
-export { ArtworkDetailsQueryRenderer as ArtworkDetails } from "reaction/Apps/Artwork/Components/ArtworkDetails"
+export {
+  ReactionArtworkArtistInfo as ArtworkArtistInfo,
+} from "./ReactionArtworkArtistInfo"
```

Comment:
I'm also seeing a lot of whitespace formatting churn, as [others have noted](https://artsy.slack.com/archives/CP9P4KR35/p1576186698265400). Haven't tracked down the source of this 🤔